.github/workflows/check-test.yaml

```diff
@@ -72,13 +72,8 @@ jobs:
           E2E_TESTS_ATLAS_PASSWORD: ${{ secrets.E2E_TESTS_ATLAS_PASSWORD }}
           # Matches what we are doing in Evergreen
           MONGODB_VERSION: '4'
-          DEBUG: 'compass-e2e-tests*'
-        run: npm run test-ci -- --stream --since $MAIN_BRANCH_NAME
+        # compass-e2e-tests are ignored as we are already running tests against
+        # a packaged application in a separate workflow, so running them here
+        # slows down things while testing the same thing
+        run: npm run test-ci -- --stream --since $MAIN_BRANCH_NAME --ignore compass-e2e-tests
```

Comment:
Want to highlight this and explain in a bit more detail: these e2e tests were passing, but required a timeout bump, I was deciding if I should just bump timeout / parallelize these particular e2e tests so they fit the current one, or just switch them off and decided to disable them as running tests against a packaged app (what we are doing in the build flow) feels like more than enough to me. Please tell me if you feel otherwise and we can discuss in what way to bring them back.

For some context the difference between this e2e test run and the one in the build workflow is that this one is testing against an application that is running production code with unpackaged electron, the one in build workflow packages the app into a fully isolated app and runs tests against it, which I think is closer to reality / better anyway.